src/wallet/params.rs

```diff
@@ -162,6 +166,13 @@ impl CreateParams {
         self
     }
 
+    /// Begin wallet scanning from the specified birthday. Particularly useful for block-based
+    /// scanning sources.
+    pub fn birthday(mut self, birthday: BlockId) -> Self {
+        self.birthday = Some(birthday);
```

Comment:
Hmm, it's kind of odd to having to always having to remember the birthday outside of the wallet and setting it on `CreateParams`. Shouldn't the wallet itself also remember its birthday, and would it need to be persisted as part of the `ChangeSet`?